app/views/products/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+ 
+   <div class="container">
+   <%= simple_form_for product do |f| %>
+  <div class="row">
+    <div class="col">
+      <%= f.input :title %>
+    </div>
+    #need to add description 
+  </div>
+  <div class="form-row">
+    <div class="form-group col-md-3">
+      <%= f.input :status %>
+    </div>
+    <div class="form-group col-md-3">
+      <%= f.input :condition %>
+ 
+      </select>
+    </div>
+    <%# <div class="form-group col-md-6">
+      <%= f.input :address %>
+    <%# </div> %>
+  </div>
+  <div class="row">
+    <div class="col">
+      <%= f.input :address %>
+    </div>
+    <div class="col">
+      <%= f.input :photo, as: :file %>
```

Comment:
I think we need to add Active Support before we can add a field for the photos. It will also need it to support multiple photos